src/main/java/com/nike/cerberus/service/CertificateService.java

```diff
@@ -584,4 +592,20 @@ private String getFileContents(final File certDir, final String filename) {
     private String getPath() {
         return "/cerberus/" + environmentMetadata.getName() + "/";
     }
+
+    /**
+     * Deletes a certificate by name from S3 and from the identity management service and from the environment config
+     *
+     * @param certificateName the certificate to delete
+     */
+    public void deleteCertificate(String certificateName) {
+        try {
+            identityManagementService.deleteServerCertificate(certificateName);
+        } catch (AmazonServiceException e) {
+            log.error("Failed to delete the certificate from the identity management service," +
```

Comment:
Should the certificateName be logged here too?   

Not sure if the name is obvious or not.

Reply:
- [ ] Log name